### PR TITLE
Ant3 upgrade

### PR DESCRIPTION
**Pull request by davidjumani**
### Description

Continuation of https://github.com/apache/cloudstack/pull/6369
This PR for Fixes https://github.com/apache/cloudstack/issues/6342. Upgrade ant-design-vue to version 3.x. Related https://www.antdv.com/docs/vue/migration-v3.

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by rohityadavcloud** [comment]
@blueorangutan package

**Comment by davidjumani** [comment]
@blueorangutan package

**Comment by rohityadavcloud** [comment]
@blueorangutan ui

**Comment by davidjumani** [comment]
@blueorangutan package

**Comment by davidjumani** [comment]
@blueorangutan test

**Comment by rohityadavcloud** [comment]
@blueorangutan package

**Review by shwstppr — APPROVED** [review]
Code change LGTM. I don't see any issue in the UI. Have asked a question regarding defining labels for select option elements which at present seems to be done twice.

File: ui/src/components/header/SamlDomainSwitcher.vue [hunk 1]
```diff
@@ -42,7 +42,7 @@
         </a-tooltip>
       </template>
 
-      <a-select-option v-for="(account, index) in samlAccounts" :key="index">
+      <a-select-option v-for="(account, index) in samlAccounts" :key="index" :label="`${account.accountName} (${account.domainName})`">
```
[the remark this answers is not in the document]
Comment (davidjumani) [reply ui/src/components/header/SamlDomainSwitcher.vue:45]: Yes, as now in antdv3 searching in dropdowns can be done either by key or label, and not based on the text in the element

**Comment by davidjumani** [comment]
@blueorangutan package

**Review by kiranchavala — APPROVED** [review]
LGTM, Performed a test plan based on #7184 and #7283

 And also cross-checked  the following scenarios 
 
     VMware with deploy-as-is templates with multiple disks and networks (register, deploy and autoscaling)

    Autoscaling

    Quotas

    Advanced zones + SG

    Basic Zone

    Backups

    Userdata

    IPv6 (firewall rules. lb, etc)

**Comment by kiranchavala** [comment]
@blueorangutan ui

**Pull request merged**